[Bug]: Fix binary-stacker Makefile target

### zot version

v2.0.0-rc6

### Describe the bug

[Building a zot container image from souce](https://zotregistry.io/v1.4.3/admin-guide/admin-getting-started/#building-a-zot-container-image-from-source) with Stacker fails:
`
midgard@joro:~/work/github.com/project-zot/zot$ make binary-stacker 
/home/midgard/work/github.com/project-zot/zot/hack/tools/bin/stacker --debug build \
	-f build/stacker.yaml \
	--substitute PWD=$PWD \
	--substitute COMMIT=$COMMIT \
	--substitute ARCH=$ARCH \
	--substitute OS=$OS
/bin/sh: 1: /home/midgard/work/github.com/project-zot/zot/hack/tools/bin/stacker: not found
make: *** [Makefile:325: binary-stacker] Error 127
`


### To reproduce

**_make binary-stacker_**

### Expected behavior

Makefile target should succeed.

### Screenshots

_No response_

### Additional context

_No response_

--- a/Makefile
+++ b/Makefile
@@ -321,13 +321,14 @@ run-container:
 		zot-build:latest 
 
 .PHONY: binary-stacker
-binary-stacker:
+binary-stacker: $(STACKER)
 	${STACKER} --debug build \
 		-f build/stacker.yaml \
-		--substitute PWD=$$PWD \
-		--substitute COMMIT=$$COMMIT \
-		--substitute ARCH=$$ARCH \
-		--substitute OS=$$OS
+		--substitute COMMIT=$(COMMIT) \
+		--substitute ARCH=$(ARCH) \
+		--substitute OS=$(OS) \
+		--substitute RELEASE_TAG=$(RELEASE_TAG) \
+		--substitute REPO_NAME=zot-$(OS)-$(ARCH)
 
 .PHONY: image
 image:

--- a/build/stacker.yaml
+++ b/build/stacker.yaml
@@ -13,7 +13,7 @@ build:
     cd /go/src/github.com/project-zot
     git clone /zotcopy zot
     cd /go/src/github.com/project-zot/zot
-    make COMMIT=${{COMMIT}} OS=${{OS}} ARCH=${{ARCH}} RELEASE_TAG=${{RELEASE_TAG}} clean binary${{EXT:}}
+    make COMMIT=${{COMMIT}} OS=${{OS}} ARCH=${{ARCH}} RELEASE_TAG=${{RELEASE_TAG}} clean binary
     cat > config.json << EOF
     {
       "storage":{
@@ -49,7 +49,7 @@ build:
 
     mkdir -p binary/ cert/ config/
 
-    cp /go/src/github.com/project-zot/zot/bin/zot-${{OS}}-${{ARCH}}${{EXT:}} binary/
+    cp /go/src/github.com/project-zot/zot/bin/zot-${{OS}}-${{ARCH}} binary/
     cp /go/src/github.com/project-zot/zot/config.json config/
     cp /etc/ssl/certs/ca-certificates.crt cert/
   build_only: true
@@ -68,7 +68,7 @@ build:
     - source: ../.build/${{REPO_NAME}}/config
       dest: /etc/zot
   entrypoint:
-    - /usr/local/bin/zot-${{OS}}-${{ARCH}}${{EXT:}}
+    - /usr/local/bin/zot-${{OS}}-${{ARCH}}
   volumes:
     - /var/lib/registry
   cmd:

